src/toggle/toggle.styles.tsx

```diff
@@ -0,0 +1,187 @@
+                        }
+                    `;
+                } else {
+                    return css`
+                        border-color: transparent;
+                    `;
+                }
+            }
+
+            default: {
+                if (props.$disabled && !props.$selected) {
+                    return css`
+                        background: ${Color.Neutral[6]};
+                        border-color: ${Color.Neutral[5]};
+                    `;
+                } else if (props.$disabled && props.$selected) {
+                    return css`
+                        background: ${Color.Neutral[6]};
+                        border-color: ${Color.Neutral[4]};
+                    `;
+                } else if (props.$error) {
+                    return css`
+                        background: ${Color.Neutral[8]};
+                        border-color: ${Color.Validation.Red.Border};
+
+                        :hover {
+                            box-shadow: 0 0 4px 1px ${Color.Shadow.Red};
+                        }
+                    `;
+                } else if (props.$selected) {
+                    return css`
+                        background: ${Color.Accent.Light[5]};
+                        border-color: ${Color.Primary};
+
+                        :hover {
+                            box-shadow: 0 0 4px 1px ${Color.Shadow.Accent};
+                        }
+                    `;
+                } else {
+                    return css`
+                        background: ${Color.Neutral[8]};
+                        border-color: ${Color.Neutral[5]};
+
+                        :hover {
+                            box-shadow: 0 0 4px 1px ${Color.Shadow.Accent};
+                        }
+                    `;
+                }
+            }
+        }
+    }}
+`;
+
+export const Input = styled.input`
+    position: absolute;
+    height: 100%;
+    width: 100%;
+    cursor: ${(props) => (props.disabled ? "not-allowed" : "pointer")};
+    top: 0;
+    left: 0;
+
+    /* Hide appearance but keep it focusable using keyboard interactions */
+    appearance: none;
+    background: transparent;
+`;
+
+export const TextContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+export const Label = styled.label<LabelStyleProps>`
+    ${(props) => {
+        if (props.$selected && !props.$indicator) {
+            return css`
+                ${TextStyleHelper.getTextStyle("H4", "semibold")}
+            `;
+        } else {
+            return css`
+                ${TextStyleHelper.getTextStyle("H4", "regular")}
+            `;
+        }
+    }}
+
+    color: ${Color.Neutral[1]};
+
+    ${(props) => {
+        if (props.$disabled) {
+            return css`
+                color: ${Color.Neutral[3](props)};
+            `;
+        } else if (props.$selected) {
+            return css`
+                color: ${Color.Primary(props)};
+            `;
+        }
```

Comment:
if indicator + disabled + selected, color is N1

Reply:
Clarified and we'll stick to N3 for now for all disabled states regardless if indicator present